release/models/isis/openconfig-isis-lsp.yang

```diff
@@ -34,7 +34,13 @@ submodule openconfig-isis-lsp {
     Section 4.c of the IETF Trust's Legal Provisions Relating
     to IETF Documents (http://trustee.ietf.org/license-info).";
 
-  oc-ext:openconfig-version "1.6.1";
+  oc-ext:openconfig-version "1.6.2";
+
+  revision "2024-02-20" {
+    description
+      "Fix typo in RFC reference for adjacency-state.";
+    reference "1.6.2.";
```

Comment:
Wait, I just noticed the trailing `.` here and in the other files -- I don't think you intended that!

Reply:
Fixed